src/examples/src/config.ts

```diff
@@ -25,6 +25,7 @@ import Username from './widgets/constrained-input/Username';
 import BasicDialog from './widgets/dialog/Basic';
 import BasicEmailInput from './widgets/email-input/Basic';
 import BasicGrid from './widgets/grid/Basic';
+import BasicForm from './widgets/form/Basic';
```

Comment:
I believe these imports should be sorted alphabetically now (as per @tomdye 's request)

Reply:
yes please, helps massively with merge conflict avoidance.